components/meter/meter-circle.js

```diff
@@ -57,26 +74,33 @@ class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 
 		const primary = this._primary(this.value, this.max) || '';
 		const secondary = this._secondary(this.value, this.max, this.text);
+		const secondaryTextElement = (this.text && !this.textHidden) ? html`<div class="d2l-body-small d2l-meter-circle-text-secondary">${secondary}</div>` : nothing;
 		const textClasses = {
 			'd2l-meter-circle-text-ltr': !this.percent,
 			'd2l-body-standard': true,
 			'd2l-meter-circle-text': true
 		};
 
 		return html`
-			<svg viewBox="0 0 48 48" shape-rendering="geometricPrecision" role="img" aria-label="${this._ariaLabel(primary, secondary)}">
-				<circle class="d2l-meter-circle-full-bar" cx="24" cy="24" r="21"></circle>
-				<circle
-					class="d2l-meter-circle-progress-bar"
-					cx="24" cy="24" r="21"
-					stroke-dasharray="${progressFill} ${space}"
-					stroke-dashoffset="${dashOffset}"
-					visibility="${visibility}"></circle>
+			<div
+				class="${(this.text && !this.textHidden) ? 'fit-content ' : ''}d2l-meter-circle"
```

Comment:
This class seems kinda weird. I don't see it specified anywhere. 🤔 

The semantics here seem a little weird - we have an `img` inside of an `img`. Is there a way to specify the aria-label on the actual image (svg), so that the div is simply used for layout?

Reply:
Well... this is just copying what `meter-radial` is already doing so they're consistent. But yes, your concerns still seem valid.

Reply:
The reason we need the "fit-content" set for when there is text to display is because the text may riun over the 2.4rem width set int he host

Reply:
Looks like in the `meter-radial` case, it doesn't specify `role="img"` on the `svg`. Not sure if there is a meaningful difference there or not.

Reply:
>The reason we need the "fit-content" set for when there is text to display is because the text may riun over the 2.4rem width set int he host

I don't see where that `fit-content` class is defined in the CSS. Maybe that was missed from checkin?

Reply:
I don't think `fit-content` is doing what you think here -- it's putting it as a `class` but there's no `fit-content` CSS class defined. You have a separate CSS rule that's being applied and is likely accomplishign this for you:

```css
:host([text]:not([text-hidden])) {
  width: fit-content;
}
```

Reply:
I agree with Dave B here -- I think the `role=img` can likely be removed from the `svg` here so that they match. But I'd want to do some accessibility testing here to make sure it's reading everything out as you'd expect.

Reply:
oh sorry my bas - yes that class assignment should have been removed for the dynamic css selector stuff